homeassistant/components/ondilo_ico/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Ondilo ICO",
```

Comment:
I don't think this title needs to be translated?

Reply:
No it doesn't. Does it mean I should remove it from the file?

Reply:
Yes, because this adds it to translations 😉

Reply:
OK Done and pushed to my branch. Should I open a new PR as this one is closed?

Reply:
Yeah, and I'm happy to get tagged to review quickly

Reply:
@allenporter Done. PR is here: https://github.com/home-assistant/core/pull/44824